recipe/meta.yaml

```diff
@@ -36,10 +36,25 @@ requirements:
     - traitlets
 
 test:
+  requires:
+    - nose
+    - setuptools
+    - wheel
+    - nose
+    - coverage
+    - nodejs
+    - phantomjs
```

Comment:
If you want to help tackle the robustness of the notebook js test suite, that would be terrific. We've been struggling with it for a few years now. Everything you said about running tests for extensions is true, and should be done in the CI for those projects. Conda-forge isn't the CI for the project, though.